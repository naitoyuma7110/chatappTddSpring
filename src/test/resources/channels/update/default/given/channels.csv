id,name
1,1つ目のチャンネル
2,2つ目のチャンネル
3,3つ目のチャンネル