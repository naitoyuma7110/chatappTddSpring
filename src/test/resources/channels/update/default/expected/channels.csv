id,name
1,1つ目のチャンネル
2,2つ目のチャンネル
3,更新された3つ目のチャンネル